Team6220_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team6220_CENTERSTAGE/Utilities.java

```diff
@@ -0,0 +1,47 @@
+package org.firstinspires.ftc.team6220_CENTERSTAGE;
+
+public class Utilities {
+    /**
+     * keeps angles between [-180,180] so that it does not exceed possible imu readings
+     * tested in https://www.desmos.com/calculator/igccxromri
+     * @param angle angle to limit (degrees)
+     * @return equivalent angle between -180 and 180
+     */
+    public static double limitAngle(double angle) {
+        return gooderMod((angle + 180.0), 360.0) - 180.0;
+    }
+    // does modulus similar to desmos mod function, normal java % is different and doesn't work
+    public static double gooderMod(double a, double b) {
```

Comment:
Name could be more descriptive. How is this different from the modulus operator (%)?